model/cli/spans.yaml

```diff
@@ -0,0 +1,14 @@
+groups:
+  - id: span.cli
+    type: attribute_group
+    stability: experimental
+    brief: >
+      Describes CLI (Command Line Interfaces) programs span.
+    attributes:
+      - ref: process.command_line
```

Comment:
Should I reference all attributes that are considered alternative in the process semconv?

Reply:
I'd prefer references to all of them.

Reply:
We should pick a reasonable set - what we do in process attributes today is really an anti-pattern  - https://github.com/open-telemetry/semantic-conventions/issues/864.

Reply:
@lmolkova 

When you say reasonable set, do you mean find a set of attributes and mark them as required?

Reply:
I mean that we have an extensive list of all `process` attributes defined in the registry - https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/process.md

We need to pick attributes that sufficiently describe the process span and set requirement levels. 

If we want to start small - I have no objections. On the first sight, I'd add the following attributes to the span:
- `process.executable.name` as required - it could be in the command line, but it's something would want to query on and group by
- `process.executable.path` - required or recommended
- `process.pid` - required (if it's always possible to obtain it)
- `process.exit.code` - required (or conditionally_required if not 0)
- `process.title` - recommended this should probably go the the span name
- `process.command_args` (array) or another one that describes command line

There are other applicable attributes, but they can be added later if anyone  needs them.

Reply:
Added this but `process.title` which I haven't find in the registry.